EnglishFilter/sections/css_extended.txt

```diff
@@ -1406,6 +1406,7 @@ lawlex.org#?#.sidebar > div > ul > .widget:has(> a:not([href^="https://lawlex.or
 medicalnewstoday.com#?#span ~ div[class]:has(> aside[class] div[data-empty="true"])
 getfreecourses.org,ftuudemy.com,freecourseslab.com,ftuforums.com,freecoursessites.com,freetutorialseu.com#?#.widget-area > aside.widget_custom_html:has(> div > ins.adsbygoogle)
 republicworld.com#?#div[class] > div[id^="div-gpt-ad"]:upward(1)
+hentaihand.com#?#.py-4 > div.row:has(> div.container > div[style="padding-bottom: 10px !important; height: 110px;"])
```

Comment:
1. Exchange this rule to the `hentaihand.com#?#main > div.row:has(> div.container iframe[src*="o333o.com/"])`.
2. Also add `hentaihand.com##main > div.row.mx-auto` - it seems ok on my end.